### ISSUE TITLE
Failed to import xgboost in Python3.5

### ISSUE THREAD

**Issue by nyanp** [dmlc/xgboost#5328]
From xgboost 1.0.0, importing xgboost in Python 3.5 causes an error because of f-strings that is not supported in 3.5.

```
/opt/hostedtoolcache/Python/3.5.9/x64/lib/python3.5/site-packages/xgboost/dask.py:33: in <module>
    from .sklearn import XGBModel, XGBClassifierBase, xgboost_model_doc
E     File "/opt/hostedtoolcache/Python/3.5.9/x64/lib/python3.5/site-packages/xgboost/sklearn.py", line 437
E       msg = f'Current model type: {type(self).__name__}, ' + \
E                                                          ^
E   SyntaxError: invalid syntax
```

- xgboost version: 1.0.0 installed from pip
- environment: ubuntu-latest in GitHub CI ([link to my CI result](https://github.com/nyanp/nyaggle/runs/457601501))

Is Python3.5 no longer supported from 1.0.0?

**Comment by hcho3** [comment]
F-string should not have been in there. We will patch it.

**Comment by hcho3** [comment]
#5330 should fix the issue.

**Comment by nyanp** [comment]
Thanks for your quick response :)

**Comment by hcho3** [comment]
I just released 1.0.1, which is identical to 1.0.0 except that 1.0.1 actually supports Python 3.5.

**Comment by trivialfis** [comment]
Score one for semantic versioning.
